crypto/certs.go

```diff
@@ -28,10 +28,10 @@ import (
 	"time"
 )
 
-func GenerateTlsCert(path string) (string, error) {
+func NewTlsCerts(path, host string) (string, string, error) {
```

Comment:
Add godoc to crypto package and methods.
It is difficult to guess the function output (string, string,...).